app/services/hyku_addons/cache_expiration_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  class CacheExpirationService
+    def perform
+      Account.all.find_each do |account|
+        Rails.logger.info "Expiring cache for #{account.cname}"
+        next if account.redis_endpoint.is_a?(NilRedisEndpoint)
+
+        expire_cache_for(account)
+      end
+    end
+
+    def expire_cache_for(account)
+      account.redis_endpoint.switch!
+      account.setup_tenant_cache(true)
```

Comment:
Could we not call account.switch! and save a line?

Reply:
There are some extra things made in `account.switch!` that we really don't need here, including loading the settings which can be costly.
This should avoid that